### DISCUSSION TITLE
剛剛問共學gpt怎麼學習python，它叫我去寫小遊戲

### DISCUSSION THREAD

**Discussion by Chengbosco** [1101-GS4719-C/Python-Community-GS4719-C#1652] [General]
剛剛問共學gpt怎麼學習python，它叫我去寫小遊戲，那他剛剛第一個推薦的是從1~99猜數字的小遊戲，於是我就寫了（用來貼稿）。
import random
rn = random.randint(1, 99)
＃print(rn)
print("你猜多少?")
a=int(input(">"))
for i in range(11):
    if i==10:
        print("十次機會已用完，再來一次?")
    elif rn>a:
        print("大於",a)
        a=int(input(">"))
    elif rn<a:
        print("小於",a)
        a=int(input(">"))
    else:
        print("對了，答案是",rn)
        break

有個小問題是有時候print("你猜多少?")會出不來，輸入的數字會延遲，兩個憶起跳出來，但我不確定是我電腦太慢還是怎樣，可以等你們試完再和我說！其實比我想像中簡單，只是要仔細一點。之前有寫過剪刀石頭布，回去再找來貼（賺分）。

**Comment by Chengbosco** [comment]
＊兩個數字＊上面打錯了

**Comment by lazywulf** [comment]
版主，markdown 會打嗎？我看了眼睛好痛。也順便練了一次喔，在此獻醜。
```python
from random import randint

rn = randint(1, 99)
for i in range(10):
    a = int(input("你猜多少?\n>"))
    if rn != a:
        print(f"{'大' if rn > a else '小'}於 {a}")
    else:
        print(f"對了，答案是 {rn}")
        break
else:
    print("十次機會已用完，再來一次?")
```
